### PR TITLE
Feature/login implement behavior

### PR DESCRIPTION
**Pull request by RachelLivne**
login-implement

### CODE REVIEW

File: src/config/config.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
-const BASE_URL = process.env.BASE_URL;
-
+// const BASE_URL = process.env.BASE_URL;
+const BASE_URL = "http://localhost:8080"
```
Comment (efef10) [inline src/config/config.tsx]: put in env file as REACT_APP_BASE_URL

File: src/pages/landingPage.tsx [hunk 2]
```diff
@@ -8,9 +8,9 @@ import Tab from '@mui/material/Tab';
 const LandingPage: React.FC = () => {
   const [value, setValue] = React.useState("pendingOrders");
   let navigater = useNavigate()
-  useEffect(() => {
```
Comment (efef10) [inline src/pages/landingPage.tsx]: are you sure you want to delete this?

File: src/config/config.tsx [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 // Define the full URL for getting all orders
 const GET_ALL_ORDERS_URL = `${BASE_URL}/orders`;
-
-export { BASE_URL,GET_ALL_ORDERS_URL };
+const Log_In =`${BASE_URL}/User/login`;
```
Comment (efef10) [inline src/config/config.tsx]: change variable name to LOGIN_URL

File: src/pages/login.tsx [hunk 4]
```diff
@@ -26,20 +31,41 @@ const Login: React.FC = () => {
   const handleMouseDownPassword = (event: React.MouseEvent<HTMLButtonElement>) => {
     event.preventDefault();
   };
+  const login = async () => {
+
+    console.log({ email }, { password })
```
Comment (efef10) [inline src/pages/login.tsx]: remove console.log

**Comment by RachelLivne** [comment]
login-implement

**Review by efef10 — APPROVED** [review]

**Pull request merged**